pkg/kubelet/volume/empty_dir/empty_dir_test.go

```diff
@@ -27,7 +27,7 @@ import (
 
 func TestCanSupport(t *testing.T) {
 	plugMgr := volume.PluginMgr{}
-	plugMgr.InitPlugins(ProbeVolumePlugins(), &volume.FakeHost{"/tmp/fake"})
+	plugMgr.InitPlugins(ProbeVolumePlugins(), &volume.FakeHost{"/tmp/fake", nil})
```

Comment:
Are these changes actually necessary? Doesn't go initialize an unspecified value to nil?